colorama/ansitowin32.py

```diff
@@ -21,6 +21,20 @@ def is_a_tty(stream):
     return hasattr(stream, 'isatty') and stream.isatty()
 
 
+def check_win_10_ansi_support():
```

Comment:
Is it possible to replace the windows release check using the "platform" module? platform.win32_ver() or platform.platform()?